app/src/test/java/uk/ac/ebi/atlas/experimentpage/cellplot/CellPlotDaoIT.java

```diff
@@ -53,28 +53,30 @@ void populateDatabaseTables() {
         populator.setScripts(
                 new ClassPathResource("fixtures/experiment.sql"),
                 new ClassPathResource("fixtures/scxa_analytics.sql"),
-                new ClassPathResource("fixtures/scxa_coords.sql"),
                 new ClassPathResource("fixtures/scxa_cell_group.sql"),
-                new ClassPathResource("fixtures/scxa_cell_group_membership.sql"));
+                new ClassPathResource("fixtures/scxa_cell_group_membership.sql"),
+                new ClassPathResource("fixtures/scxa_dimension_reduction.sql"),
+                new ClassPathResource("fixtures/scxa_coords.sql"));
         populator.execute(dataSource);
     }
 
     @AfterAll
     void cleanDatabaseTables() {
         populator.setScripts(
-                new ClassPathResource("fixtures/scxa_cell_group_marker_gene_stats-delete.sql"),
-                new ClassPathResource("fixtures/scxa_cell_group_marker_genes-delete.sql"),
+                new ClassPathResource("fixtures/scxa_coords-delete.sql"),
```

Comment:
Please swap these two tables as we use `scxa_dimension_reduction` table foreign key reference in the `scxa_coords` table. I think it will throw errors while cleaning tables.

Reply:
As below, deletions should be done in the inverse order of insertions, precisely to avoid removing foreign keys. In any case I think Spring puts everything under a single transaction and it’s not important.